src/app/components/genericLabel/Label.module.scss

```diff
@@ -0,0 +1,45 @@
+@use '../../../styles/colors';
+@use '../../../styles/common';
+@use '../../../styles/typography';
+
+.labelwrapper{

```

Comment:
Ustaliliśmy, że przy nazwach klas używamy camelCase :) Dotyczy to również pozostałych klas.

Reply:
Poprawione!